core/src/test/resources/druid-broker.properties

```diff
@@ -1,23 +1,30 @@
 #
-# Tranquility.
-# Copyright 2013, 2014, 2015  Metamarkets Group, Inc.
+# Licensed to Metamarkets Group Inc. (Metamarkets) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  Metamarkets licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
+#   http://www.apache.org/licenses/LICENSE-2.0
 #
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
 #
 
 druid.host=localhost
 druid.port=:DRUIDPORT:
 druid.service=druid:tranquility:broker
+
 druid.zk.service.host=:ZKCONNECT:
+
 druid.processing.numThreads=1
 druid.processing.buffer.sizeBytes=8388608
+
+druid.selectors.indexing.serviceName=druid:tranquility:indexer
```

Comment:
I needed to add a coordinator since 0.8.2 overlords are more finicky about wanting to create their DB tables, and the coordinator by default is the derby metadata server (in 0.8.1 it failed & continued).

Along with adding a coordinator I took the opportunity to update all the runtime properties files.